C9S stream 3.11 is failing here:
```
[PASSED] for 'locale-test-app' test_application_enable_init_wrapper (00:00:04)
 [FAILED] for 'pipenv-test-app' run_s2i_build ()
 [PASSED] for 'pipenv-and-micropipenv-should-fail-test-app' run_s2i_build ()
```

```
[notice] A new release of pip available: 22.3.1 -> 26.1.2
[notice] To update, run: python3.11 -m pip install --upgrade pip
---> Installing dependencies via pipenv ...
Courtesy Notice:
Pipenv found itself running within a virtual environment,  so it will 
automatically use that environment, instead of  creating its own for any 
project. You can set
PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and 
create  its own instead.
You can set PIPENV_VERBOSITY=-1 to suppress this warning.
Installing dependencies from Pipfile.lock (63c7a0)...
Looking in indexes: https://pypi.python.org/simple
Obtaining file:///opt/app-root/src (from -r 
/tmp/pipenv-5798bq3s-requirements/pipenv-_pr2vb_8-reqs.txt (line 1))
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Checking if build backend supports build_editable: started
  Checking if build backend supports build_editable: finished with status 'done'
  Getting requirements to build editable: started
  Getting requirements to build editable: finished with status 'done'
ERROR: Exception:
Traceback (most recent call last):
  File 
"/opt/app-root/src/.local/venvs/pipenv/lib/python3.11/site-packages/pipenv/patch
ed/pip/_internal/cli/base_command.py", line 109, in _run_wrapper
    status = _inner_run()
             ^^^^^^^^^^^^
  File 
"/opt/app-root/src/.local/venvs/pipenv/lib/python3.11/site-packages/pip
```

--- a/examples/mod-wsgi-test-app/app.py
+++ b/examples/mod-wsgi-test-app/app.py
@@ -1,7 +1,10 @@
 import os
-import mod_wsgi.server
+try:
+    import mod_wsgi.express.cli as server_wrapper
+except ImportError:
+  import mod_wsgi.server as server_wrapper
 
-mod_wsgi.server.start(
+server_wrapper.start(
   '--log-to-terminal',
   '--port', '8080',
   '--trust-proxy-header', 'X-Forwarded-For',

